sdk/src/main/java/com/uid2/utils/TimeUtils.kt

```diff
@@ -5,6 +5,11 @@ package com.uid2.utils
  */
 internal class TimeUtils {
 
+    /**
+     * Returns the current epoch time (in milliseconds).
+     */
+    fun now() = System.currentTimeMillis()
```

Comment:
This should be `@JvmStatic`?

Reply:
This is in a class (which is mocked) so can't be `@JvmStatic`, right? As this current version allows me to control what `now` is within the UT.

Reply:
sgtm